srecord/arglex.h

```diff
@@ -150,7 +150,7 @@ class arglex
       * name, and turn it into the minimum possible string for that
       * option.
       */
-    static auto abbreviate(const char *text) -> std::string;
+    static auto abbreviate(const char *s) -> std::string;
```

Comment:
I like this check however, it appears that in a large number of cases, the more meaningful name is the one in the header whereas the correction is making use of that in the implementation. Keen to leave in the check, however the fix may need to be more manual to get the best result.